altos-rust/altos-core/src/task/control.rs

```diff
@@ -596,14 +591,14 @@ mod tests {
     use super::*;
     use test;
 
-    fn get_task() -> TaskControl {
+    fn test_get_task() -> TaskControl {
```

Comment:
These appear to be helper functions, not tests themselves. Changing the name has caused compiler errors because these functions are being called by a bunch of the other tests. It might make sense to move this out to test.rs, or if it's left here, maybe add an underscore to the name like `_get_task`. Either way, all the other test functions that are calling this should be updated.